runtime/nls/shrc/j9shr.nls

```diff
@@ -6896,3 +6896,24 @@ J9NLS_SHRC_CM_PRINTSTATS_PROCESSOR_FEATURES.explanation=NOTAG
 J9NLS_SHRC_CM_PRINTSTATS_PROCESSOR_FEATURES.system_action=
 J9NLS_SHRC_CM_PRINTSTATS_PROCESSOR_FEATURES.user_response=
 # END NON-TRANSLATABLE
+
+J9NLS_CC_MSYNC_UNSUPPORTED=The system does not support synchronizing memory with physical storage
+# START NON-TRANSLATABLE
+J9NLS_CC_MSYNC_UNSUPPORTED.explanation=Synchronization between memory and physical storage is not supported on the runtime platform, so the feature is disabled.
+J9NLS_CC_MSYNC_UNSUPPORTED.system_action=The JVM won't force the operating system to flush changes in memory to the physical storage.
+J9NLS_CC_MSYNC_UNSUPPORTED.user_response=No action required.
+# END NON-TRANSLATABLE
+
+J9NLS_SHRC_SHRINIT_HELPTEXT_MAP64=Use 64-bit mapping to map the shared cache above the 2G addressing range if it is supported.
+# START NON-TRANSLATABLE
+J9NLS_SHRC_SHRINIT_HELPTEXT_MAP64.explanation=NOTAG
+J9NLS_SHRC_SHRINIT_HELPTEXT_MAP64.system_action=
+J9NLS_SHRC_SHRINIT_HELPTEXT_MAP64.user_response=
+# END NON-TRANSLATABLE
+
+J9NLS_SHRC_OSCACHE_MMAP_MSYNC_ERROR=Error Synchronizing memory
+# START NON-TRANSLATABLE
+J9NLS_SHRC_OSCACHE_MMAP_MSYNC_ERROR.explanation=An error has occurred in shared class processing.
+J9NLS_SHRC_OSCACHE_MMAP_MSYNC_ERROR.system_action=The JVM terminates, unless you have specified the nonfatal option with "-Xshareclasses:nonfatal", in which case the JVM continues without using Shared Classes.
```

Comment:
I think the first occurrence of "nonfatal" should be hyphenated, i.e. "non-fatal" and "Shared Classes" need not be capitalized.

Reply:
Fixed.